proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -196,9 +191,49 @@ message VscSendTimestamp {
   google.protobuf.Timestamp timestamp = 2 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
 }
 
+//
+// Key assignment section
+//
+
+message ConsumerConsAddress {
```

Comment:
An edgecase crossed my mind:
What if the address format is different on provider and consumer?

eg. `cosmosvalconsjdfp0gks0dr9gs` vs `stridevalconsjdfp0gks0dr9gs`

Do we have/need any special handling in that case?

Reply:
I don't believe this is an issue, but I'm not 100% sure. These are consensus addresses that are persisted as bytes. `ConsumerCoonsAddress` is just a type wrapper for a byte array